Hymns/Display/DisplayHymnToolbarView.swift

```diff
@@ -0,0 +1,52 @@
+import SwiftUI
+
+struct DisplayHymnToolbarView: View {
+    @State var toolbarTab = "lyrics"
+    @ObservedObject private var viewModel: DisplayHymnViewModel
+
+    init(viewModel: DisplayHymnViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        VStack {
+            HStack {
+                Spacer()
+                Text("LYRICS").foregroundColor(self.toolbarTab == "lyrics" ? .accentColor : .primary).onTapGesture {
+                    self.toolbarTab = "lyrics"
+                }
+                Spacer()
+                Text("CHORDS").foregroundColor(self.toolbarTab == "chords" ? .accentColor : .primary).onTapGesture {
+                    self.toolbarTab = "chords"
+                }
+                Spacer()
+                Text("GUITAR").foregroundColor(self.toolbarTab == "guitar" ? .accentColor : .primary).onTapGesture {
+                    self.toolbarTab = "guitar"
+                }
+                Spacer()
+                Text("PIANO").foregroundColor(self.toolbarTab == "piano" ? .accentColor : .primary).onTapGesture {
+                    self.toolbarTab = "piano"
+                }
+                Spacer()
+            }
+            Divider().edgesIgnoringSafeArea(.horizontal)
+            if self.toolbarTab == "lyrics" {
+                HymnLyricsView(viewModel: self.viewModel.hymnLyricsViewModel).padding(.horizontal)
+            } else if self.toolbarTab == "chords" {
+                // Text("\(self.viewModel.identifier.hymnNumber)")
+                // print("\(self.viewModel.identifier.hymnNumber)")
+                WebView(request: URLRequest(url: URL(string: "https://www.hymnal.net/en/hymn/\(self.viewModel.identifier.hymnType.abbreviatedValue)/\(self.viewModel.identifier.hymnNumber)/f=gtpdf")!))
```

Comment:
So you have, in the hymnEntity object, the urls for each of the sheet music (both svg and pdf). I think it's probably better to use that than to hardcode the urls here

Reply:
Outdated